.github/workflows/ci.yml

```
@@ -1,345 +1,112 @@
-          path: |
-            coverage.xml
-            htmlcov/
-
-      - name: Comment coverage on PR
-        if: github.event_name == 'pull_request'
-        uses: py-cov-action/python-coverage-comment-action@v3
+          pip install -r requirements.txt
+      - name: Run tests with coverage
+        run: pytest tests/ --cov=rainmaker_orchestrator --cov=api --cov-report=xml --cov-report=term
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.10)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)